src/Caching/StackExchangeRedis/src/RedisCache.cs

```diff
@@ -194,10 +194,12 @@ private void Connect()
                     if (_options.ConfigurationOptions != null)
                     {
                         _connection = ConnectionMultiplexer.Connect(_options.ConfigurationOptions);
+                        _connection.RegisterProfiler(_options.ProfilingSession);
```

Comment:
Given this is not a required option, let's check for null before calling `RegisterProfilier`

Reply:
Thanks, `StackExchange.Redis` is doing null check when using it but make sense to do it here also.
Done!